### PR TITLE
update Readme.md with desktop instructions.

### PR DESCRIPTION
**Pull request by matyasf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,25 @@ by communicating with the Linux kernel through `/dev/spidev*`. This means:
...
+
+Note that the cython code will not run on Windows.    
+
+With pip only install `pillow`, do not install `RPi.GPIO` (it is only for the Pi, on desktop Linux it will just exit with an error).
+
+Then install some dependencies:
+
+```
+sudo apt-get install python3-dev
+sudo apt-get install python3-tk
```
Comment (FutureSharks) [inline README.md:58]: This dependency has been removed now unless you want to run the tests

File: README.md [hunk 2]
```diff
@@ -44,3 +44,25 @@ by communicating with the Linux kernel through `/dev/spidev*`. This means:
 
 If you modify `spi.pyx`, make sure to set the `USE_CYTHON` environment variable before building---otherwise your
 changes will not be compiled into `spi.c`.
+
+#### Running the code on Linux desktop
+
+Note that the cython code will not run on Windows.
```
Comment (FutureSharks) [inline README.md:50]:
```suggestion
Note that the python code will not run on Windows.    
```

File: README.md (hunk 1 above)
Comment (matyasf) [reply README.md:58]: The whole point of running it on desktop is to run in virtual display mode (that uses tk). I'll add some more clarification

File: README.md (hunk 2 above)
Comment (matyasf) [reply README.md:50]: again, I'll add more clarification; the python code works on Windows, the Cython code uses some calls into the Linux system thus you cannot compile it.

File: README.md (hunk 1 above)
Comment (matyasf) [reply README.md:60]: done

File: README.md (hunk 2 above)
Comment (FutureSharks) [reply README.md]: OK sorry, I thought it was a typo! I've heard of CPython but never cythong 😅

**Comment by GregDMeyer** [comment]
Thank you!! :-)

**Pull request merged**